# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1379957471 14400
#      Mon Sep 23 13:31:11 2013 -0400
# Node ID fd01c78caa0b24920f7359e3ecfac32e51d67e41
# Parent  cc13924a4266fb0359f59fabdce11071e6051d48
Added tag ss-3-7-7 for changeset cc13924a4266

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -74,8 +74,9 @@ 858cbf6fc2ec1c232f5cf1d75dc344439b39a89c
 faefa1bea8ddae3cab170afdeab68d3d15c4e623 ss-3-7-2
 0000000000000000000000000000000000000000 ss-3-7-2
 23a7661e529ae9bfc91693618f8c314c31f695ca ss-3-7-2
 cc5a7d1233f3acea85648baeb754fc0e8f225225 rc-3-6-4-2
 b29b10fbb7448cdfe29322446e1a589e7fe1a40a release-3-6-4
 4e50bd2946d8563d3e201cc04b3ba0720c991b06 ss-3-7-4
 608e307b49149b32a6d09c2f06493d04d3af9be4 ss-3-7-5
 3a9efb68272df556dccb84c87933dd8238e88902 ss-3-7-6
+cc13924a4266fb0359f59fabdce11071e6051d48 ss-3-7-7
